zadanie3/zadanie3.cpp

```diff
@@ -67,6 +67,7 @@ bool usun_element(Lista* l, Element* usun)
 
 	if(usun == l->glowa) {
 		l->glowa = e->next;
+        usun->next = NULL;
```

Comment:
to jest potrzebne  tutaj i w linii 79 bo dodalem destruktor struktury Element, ktory usuwa 'next' - jakby to nie bylo wynullowane to by destruktor mogl usuwac losowe elementy na ktore wskazuje ten wskaznik